packages/remediations/src/RemediationButton/RemediationButton.js

```diff
@@ -42,14 +42,16 @@ class RemediationButton extends React.Component {
     }
 
     render() {
+        const { children, ...buttonProps } = this.props;
```

Comment:
This can selects props that do not belong to the button such as 

```jsx
    dataProvider: propTypes.func.isRequired,
    onRemediationCreated: propTypes.func,
```

Maybe add a new prop 'buttonProps: PropTypes.object' and use it.